ui/src/components/entity/SimpleSymbolGloss.tsx

```diff
@@ -168,23 +169,25 @@ class SimpleSymbolGloss extends React.PureComponent<Props, State> {
               )}
               {this.props.showDrawerActions && (
                 <React.Fragment>
-                  <td>
-                    <MuiTooltip
-                      title={
-                        defsAndNicknames.length > 0
-                          ? `See ${defsAndNicknames.length} definitions`
-                          : "No definitions."
-                      }
-                    >
-                      <IconButton
-                        size="small"
-                        disabled={defsAndNicknames.length === 0}
-                        onClick={this.onClickDefinitionsButton}
+                  {definitionsEnabled && (
+                    <td>
```

Comment:
Can this flag change during the life cycle of this component? If so, then it's worth adding a `key` to this `<td/>` and it's peers. Makes the second render more efficient.